api/src/main/java/org/pmiops/workbench/institution/InstitutionMapper.java

```diff
@@ -50,4 +55,15 @@
         .map(address -> new DbInstitutionEmailAddress().setEmailAddress(address))
         .collect(Collectors.toSet());
   }
+
+  @AfterMapping
+  default void setUserInstruction(
+      @MappingTarget Institution target, @Context InstitutionService institutionService) {
+    institutionService
+        .getInstitutionUserInstructions(target.getShortName())
+        .ifPresent(
```

Comment:
this can be `ifPresent(target::setUserInstructions)`

Reply:
yes thats much neater thanks